reciprocalspaceship/utils/rfree.py

```diff
@@ -15,8 +15,12 @@ def add_rfree(dataset, fraction=0.05, bins=20, inplace=False):
         Dataset object for which to compute a random fraction.
     fraction : float, optional
         Fraction of reflections to be added to the r-free. (the default is 0.05)
-    bins : int, optional
-        Number of resolution bins to divide the free reflections over. (the default is 20)
+    ccp4_conventiion: bool, optional
+        Default False to use Phenix(CNS/XPLOR) convention:
+        1 is test set, 0 is working set, and key is "R-free-flags".
+        Set to True to use ccp4 convention:
+        0 is test set, 1 is working set, and the key is "FreeR_flag".
+        See https://www.ccp4.ac.uk/html/freerflag.html#description for convention details.
```

Comment:
I find this description a bit confusing -- do you mind first defining it: "Whether to use the CCP4 convention for R-free flags (Default: False). If True, use CCP4 convention... If False, use Phenix convention... "

Reply:
I will make the change.